docs/source/private-data/private-data.md

```diff
@@ -151,6 +151,15 @@ documentation on [transaction flow](../txflow.html).
    their copy of the private state database and private writeset storage. The
    private data is then deleted from the `transient data store`.
 
+The [Fabric Gateway](../gateway.html) takes care of collecting the required set of endorsements
+on behalf of the client application.  It does this by running the first simulation on a peer in
+the gateway's organization and aggregating the relevant endorsement policies based on the read/write
+set generated by the chaincode. Client applications that submit transaction requests through
+the gateway using one of the [gateway SDKs](../gateway.html#software-development-kits-sdks)
+will benefit from this [endorser selection logic](../gateway.html#how-the-gateway-endorses-your-transaction-proposal).
```

Comment:
will benefit from this endorser selection logic >> use the [endorsement logic] encoded in the gateway peer. ... (the target linked topic is not only about "endorser selection" right).